cluster-operator/src/main/java/io/strimzi/operator/cluster/model/KafkaBrokerConfigurationBuilder.java

```diff
@@ -63,6 +63,17 @@ public class KafkaBrokerConfigurationBuilder {
     private final StringWriter stringWriter = new StringWriter();
     private final PrintWriter writer = new PrintWriter(stringWriter);
     private final Reconciliation reconciliation;
+
+    /**
+     * The configuration field name for Kafka metric reporters.
+     */
+    public static final String KAFKA_METRIC_REPORTERS_CONFIG_FIELD = "metric.reporters";
```

Comment:
This is a Kafka configuration, so it should be in the `KafkaCluster` model.

Reply:
Of course. Moved to `cluster-operator/src/main/java/io/strimzi/operator/cluster/model/KafkaCluster.java
`